src/components/SignUp.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+  <div class="sign-up">
+    <p>Let's create a new account !</p>
+    <input type="text" v-model="email" placeholder="Email" /><br />
+    <input type="password" v-model="password" placeholder="Password" /><br />
+    <button @click="signUp">Sign Up</button>
+    <span>or go back to <router-link to="/login">login</router-link>.</span>
+  </div>
+</template>
+
+<script>
+import firebase from "firebase";
+export default {
+  name: "signUp",
+  data() {
+    return {
+      email: "",
+      password: "",
+    };
+  },
+  methods: {
+    signUp: function() {
```

Comment:
@aboubakar20 Please use a consistent syntax throughout the project. Either use `signup: function(){}` or `signup(){}`.

Reply:
Done